### ISSUE TITLE
Running PHP 7.1 and I get the error: PHP needs to be a minimum version of PHP 5.6.0

### ISSUE THREAD

**Issue by vephinx** [akalongman/sublimetext-codeformatter#286]
I get the error:

```
CodeFormatter

Format error:
PHP needs to be a minimum version of PHP 5.6.0
```

Packages/User/Preferences.sublime-settings:

```
"codeformatter_php_options": {
    "syntaxes": "php", // Syntax names which must process PHP formatter
    "php_path": "/usr/local/opt/php71/bin/php", // Path for PHP executable, e.g. "/usr/lib/php" or "C:/Program Files/PHP/php.exe". If empty, uses command "php" from system environments
    "format_on_save": false, // Format on save. Either a boolean (true/false) or a string regexp tested on filename. Example : "^((?!.min.|vendor).)*$"
    "php55_compat": false, // PHP 5.5 compatible mode
    "psr1": false, // Activate PSR1 style
    "psr1_naming": false, // Activate PSR1 style - Section 3 and 4.3 - Class and method names case
    "psr2": true, // Activate PSR2 style
    "indent_with_space": 4, // Use spaces instead of tabs for indentation
    "enable_auto_align": true, // Enable auto align of = and =>
    "visibility_order": true, // Fixes visibility order for method in classes - PSR-2 4.2
    "smart_linebreak_after_curly": true, // Convert multistatement blocks into multiline blocks

    // Enable specific transformations. Example: ["ConvertOpenTagWithEcho", "PrettyPrintDocBlocks"]
    // You can list all available transformations from command palette: CodeFormatter: Show PHP Transformations
    "passes": [],

    // Disable specific transformations
    "exclude": []
},
```

Terminal:

```
$ /usr/local/opt/php71/bin/php -version
PHP 7.1.0RC3 (cli) (built: Oct 15 2016 16:35:55) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.1.0-dev, Copyright (c) 1998-2016 Zend Technologies
```

What is going wrong? Thanks!

**Comment by subins2000** [comment]
Can you please provide the output of :
```bash
/usr/local/opt/php71/bin/php -r '<?php var_dump(phpversion());?>'
```

**Comment by nissicreative** [comment]
Similar issues here, and a very recent development. I have made no changes to my system that I can think of. If I leave the `php_path` key as an empty string, I get the same error as @vephinx. If I specify `"php_path": "/usr/local/opt/php71/bin/php”`I get:

```
Format error:
PHP Notice:  Undefined index: _ in phar:///Users/mike/Library/Application Support/Sublime Text 3/Packages/CodeFormatter/codeformatter/lib/phpbeautifier/fmt.phar/phpfmt.php on line 3448
```

The notice is then repeated a bunch of times in the alert dialog. I haven’t changed anything on my system, or any preferences (that I know of), and it is happening on two different machines. I’m running PHP 7.1 installed via Homebrew on OSX Yosemite.

My terminal output:

```bash
$ /usr/local/opt/php71/bin/php -r 'var_dump(phpversion());'
string(5) "7.1.1"
```

**Comment by nissicreative** [comment]
Update: I **uninstalled** the plugin via Package Control and tried manually installing an older version with the same results. I then installed the **phpfmt** plugin and got it working. I deleted CodeFormatter again, re-installed via Package Control, and still got the same errors.

I deleted my user settings (stored in Dropbox), copied the default settings over, and now it works. Hallelujah! :) I'm not sure what actually did the trick, because I had tried deleting my user settings before, Oh well, I'm happy now!
